mldsa/polyvec.h

```diff
@@ -17,8 +17,29 @@ typedef struct
 } polyvecl;
 
 #define polyvecl_uniform_eta MLD_NAMESPACE(polyvecl_uniform_eta)
+/*************************************************
+ * Name:        polyvecl_uniform_eta
+ *
+ * Description: Sample vector of polynomials with uniformly random coefficients
+ *              in [-MLDSA_ETA, MLDSA_ETA] by performing rejection sampling on
+ *              the output stream from SHAKE256(seed|nonce). The vector has
+ *              MLDSA_L polynomials.
+ *
+ * Arguments:   - polyvecl *v: pointer to output polynomial vector
+ *              - const uint8_t seed[]: byte array with seed of length
+ *                MLDSA_CRHBYTES
+ *              - uint16_t nonce: 2-byte nonce
+ **************************************************/
 void polyvecl_uniform_eta(polyvecl *v, const uint8_t seed[MLDSA_CRHBYTES],
-                          uint16_t nonce);
+                          uint16_t nonce)
+__contract__(
+  requires(memory_no_alias(v, sizeof(polyvecl)))
+  requires(memory_no_alias(seed, MLDSA_CRHBYTES))
+  requires(nonce == 0)
```

Comment:
@jakemas @mkannwischer What is the point of this parameter if it is always `0`?

Reply:
For uniformity with `polyveck_uniform_eta`, and reducing divergence from the ref implementation, let's keep it for now.

Reply:
done [62c127a](https://github.com/pq-code-package/mldsa-native/pull/229/commits/62c127a3e1203212410c345a2a381e8dcb630a29)

Reply:
ok, done [da5d51c](https://github.com/pq-code-package/mldsa-native/pull/229/commits/da5d51cec7e3a76d790f7e75042cb6c0596ccbcb)